Issue #10 - feat:dispute

Thoughtscript opened the pull request:
Hides buttons when zero fees - displays "No contractual fees remain".

For issue: https://github.com/kleros/arbitrable-payment-front/issues/10

Thoughtscript committed:
Merge

Thoughtscript committed:
feat:dispute and pay buttons hidden when 0 fees

n1c01a5 committed:
Merge branch 'develop' into feat/dispute

n1c01a5 commented on src/containers/contract/index.js:
```diff
@@ -132,9 +138,10 @@ class Contract extends PureComponent {
                   <div className="description Contract-content-item">{contract.data.description}</div>
                   {contract.data.status !== 4 && !contract.data.partyAFee && !contract.data.partyBFee ?
```
It would be good to create a `const DISPUTE_RESOLVED = 4` instead `4`

Thoughtscript committed:
fix:correct check condition

Per: https://github.com/kleros/kleros-api/blob/develop/src/constants/contract.js - I believe the check should block display of dispute button only if the status is `4` or contract `resolved`.

Thoughtscript committed:
fix:support amount checking

Support for contract amount

n1c01a5 committed:
Merge branch 'develop' into feat/dispute

n1c01a5 commented on src/containers/contract/index.js:
```diff
@@ -132,9 +138,10 @@ class Contract extends PureComponent {
+                      <div style={this.hideEmptyContractEl(contract)} className="Contract-content-actions-button Contract-actions-button-left" onClick={this.createDispute}>Create dispute</div>
+                      {contract.data.partyA === accounts.data[0] && <div style={this.hideEmptyContractEl(contract)} className="Contract-content-actions-button Contract-content-actions-button-right" onClick={this.createPay}>Pay</div>}
+                      {contract.data.partyB === accounts.data[0] && <div style={this.hideEmptyContractEl(contract)} className="Contract-content-actions-button Contract-content-actions-button-right" onClick={this.createReimburse}>Reimburse</div>}
+                      {this.showEmptyContractEl(contract) && <div className="Contract-content-item">No contractual fees remain</div>}
```
the message is more "The contract has ended". 
Ideally it would be good to have the contract status as:
- Bob pays Alice
- Bob refunds Alice
- Bob wins the dispute
- Bob made a timeout
